setup.py

```diff
@@ -53,11 +53,11 @@
     'scipy>=0.14.0',
     'seaborn>=0.7.1',
     'pandas-datareader>=0.2',
-    'empyrical>=0.2.2'
+    'empyrical>=0.3.0'
 ]
 
```

Comment:
My instinct is to allow a range. Otherwise every environment/application that uses pyfolio must use exactly this version of empyrical. This means we couldn't install zipline with a newer version of empyrical on our research platform, if we wanted newer empyrical features. If we can trust the maintainers of empyrical to keep its API backwards compatible, then we should be ok 😉 .

I'm not sure what the pandas issues were, though, so I might be off base here.